TombEngine/Game/effects/effects.cpp

```diff
@@ -1705,3 +1705,42 @@ void ProcessEffects(ItemInfo* item)
 	if (item->IsLara() && GetLaraInfo(item)->Control.WaterStatus == WaterStatus::FlyCheat)
 		item->Effect.Type = EffectType::None;
 }
+
+void TriggerAttackFlame(int x, int y, int z, Vector3 color, int size)
```

Comment:
- Use `Vector3` container. Effects shouldn't need integer positions, this is a Core relic. Later we can refactor particle structs to take advantage of this.
- Make parameters read-only references.

`void TriggerAttackFlame(const Vector3& pos, const Vector3& color, int size)`